composeApp/src/commonMain/kotlin/org/ooni/probe/ui/navigation/Navigation.kt

```diff
@@ -56,6 +57,9 @@ fun Navigation(
                         goToSettingsForCategory = {
                             navController.navigate(Screen.SettingsCategory(it).route)
                         },
+                        sendSupportEmail = {
+                            // TODO(norbel): Send support email
```

Comment:
Maybe we can use the `launchUrl` function as well, but with `mailto:email@example.org`?

Reply:
Let me update that. I was thinking of a way to add the mail template ( version, device model and other info)

Reply:
You can add `cc, bcc, subject and body` as query parameters in the url.